src/components/layout/navbar-items.module.css

```diff
@@ -1,3 +1,7 @@
 .navLinkHover:hover {
     background-color: var(--mantine-color-blue-9);
 }
+
+.navLinkProfileHover:hover {
+    background-color: var(--mantine-color-blue-8);
```

Comment:
I wonder if all the hover colors should be the same color, again worth checking with UX to confirm, i dont imagine we want two separate colors on hover in the navbar 🤔